pages/api/auth/[...nextauth].ts

```diff
@@ -86,9 +86,7 @@ export default async function auth(req: any, res: any) {
 		},
 
 		// A database is optional, but required to persist accounts in a database
-		adapter: MongoDBAdapter({
-			db: (await clientPromise).db('witness'),
-		}),
+		adapter: MongoDBAdapter(clientPromise),
```

Comment:
@Ace314159 how does it know which db to use now?

Reply:
idk but it still works lol. They don't specify in their examples either. https://github.com/nextauthjs/adapters/tree/main/packages/mongodb

Reply:
yeah ur right they simplified it here https://github.com/nextauthjs/adapters/pull/323